src/inter-aut.cc

```diff
@@ -0,0 +1,410 @@
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::STATE};
+        } else if (is_naming_enum(mata.node_naming) && contains(mata.nodes_names, token)) {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::NODE};
+        } else if (is_naming_enum(mata.symbol_naming) && contains(mata.symbols_names, token)) {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::SYM};
+        } else if (is_naming_marker(mata.state_naming) && token[0] == 'q') {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token.substr(1),
+                                     Mata::FormulaNode::OperandType::STATE};
+        } else if (is_naming_marker(mata.node_naming) && token[0] == 'n') {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token.substr(1),
+                                     Mata::FormulaNode::OperandType::NODE};
+        } else if (is_naming_marker(mata.symbol_naming) && token[0] == 'a') {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token.substr(1),
+                                     Mata::FormulaNode::OperandType::SYM};
+        } else if (is_naming_auto(mata.state_naming)) {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::STATE};
+        } else if (is_naming_auto(mata.node_naming)) {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::NODE};
+        } else if (is_naming_auto(mata.symbol_naming)) {
+            return Mata::FormulaNode{Mata::FormulaNode::Type::OPERAND, token, token,
+                                     Mata::FormulaNode::OperandType::SYM};
+        }
+
+        throw std::runtime_error("Unknown token " + token);
+        assert(false);
+    }
+
+    bool lower_precedence(Mata::FormulaNode::OperatorType op1, Mata::FormulaNode::OperatorType op2) {
+        if (op1 == Mata::FormulaNode::NEG) {
+            return false;
+        } else if (op1 == Mata::FormulaNode::AND && op2 != Mata::FormulaNode::NEG) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Transforms a series of tokes to postfix notation.
+     * @param aut Automaton for which transition formula is parsed
+     * @param tokens Series of tokens representing transition formula parsed from the input text
+     * @return A postfix notation for input
+     */
+    std::vector<Mata::FormulaNode> infix2postfix(Mata::InterAutomaton &aut, const std::vector<std::string> &tokens) {
```

Comment:
Input automaton should be `const`.
```suggestion
    std::vector<Mata::FormulaNode> infix2postfix(const Mata::InterAutomaton &aut, const std::vector<std::string> &tokens) {
```

Reply:
agree